### ISSUE TITLE
NUCB1 human annotations (unknown protein)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1695]
I'm assessing fission yeast unknown proteins.
We have  an ortholog of NUCB1
http://www.uniprot.org/uniprot/Q02818

NUCB1 has quite a lot of annotation, but I can't really find anything experimental except that it  appears to be Golgi (or ER localized).

- [x]  DNA binding  TAS ProtINC
No ref and seems unlikely to be true as it doesn't seem to be nuclear.

There is also a  comment"Discovered as DNA-binding protein in the serum of lupus-prone mice." but this is also unsupported.

@pgaudet  

Other TAS from Reactome include 
- [ ] cellular protein metabolic process Source: Reactome
- [ ] post-translational protein modification Source: Reactome
But I can't work out what these come from
@deustp01

**Comment by ValWood** [comment]
DNA binding also on 
http://www.uniprot.org/uniprot/P80303
http://www.uniprot.org/uniprot/P80303

**Comment by pgaudet** [comment]
Hi Val, 

There is a reference for the DNA binding annotation: 
http://europepmc.org/abstract/MED/1520323
It's an old paper but they seem to show DNA binding. Can you please check ? 

Pascale

**Comment by pgaudet** [comment]
P80303 also has a predicted DNA binding domain, see UniProt entry and PMID:7811391 (in that case though TAS is not appropriate). 

Pascale

**Comment by pgaudet** [comment]
@ValWood I forgot to tag you; did you see my response ?

**Comment by ValWood** [comment]
Hmmph, I thought I typed a long reply. I don't know what I did with it....

**Comment by ValWood** [comment]
so the only evidence for DNA binding is

one fig (the only experiment in this paper!)
![assay](https://user-images.githubusercontent.com/7359272/33201903-576b9a32-d0f2-11e7-9c0c-2f180750c3bd.jpg)

I can't see anything else that says it is not a resident Golgi protein and no reports of nuclear localization in any species.

It is unsurprizing that it binds DNA out of context, it is very basic aa composition.

It does not have a  recognizable DNA binding domain (just a coiled coil region which is  > leucine zipper  motif).

It appears to be a  type of "chaperone"

Resident golgi chaperone-like amyloid binding proteins (CLABPs)
https://www.ncbi.nlm.nih.gov/pubmed/28220836

Chaperone-like Activity of Calnuc Prevents Amyloid Aggregation.
Kanuru M, Aradhyam GK.
PMID: 27997158

Calnuc, an EF-hand Ca(2+)-binding protein, is stored and processed in the Golgi and secreted by the constitutive-like pathway in AtT20 cells.
Mol Endocrinol. 2002 Nov;16(11):2462-74.

Golgi retention of human protein NEFA is mediated by its N-terminal Leu/Ile-rich region.
PMID: 11749975 

CALNUC (nucleobindin) is localized in the Golgi apparatus in insect cells.
PMID: 10777113


The mammalian calcium-binding protein, nucleobindin (CALNUC), is a Golgi resident protein.
Lin P, Le-Niculescu H, Hofmeister R, McCaffery JM, Jin M, Hennemann H, McQuistan T, De Vries L, Farquhar MG.
J Cell Biol. 1998 Jun 29;141(7):1515-27.
PMID: 9647645

**Comment by pgaudet** [comment]
Thanks for the thorough explanation. I removed the PINC annotation from the human protein.

Pascale

**Comment by pgaudet** [comment]
Pinging @deustp01 regarding Reactome annotations

**Comment by deustp01** [comment]
Reactome: On the basis of data reported by Tagliabracci et al. (PMID: 26091039) we created a reaction in which a vast array of substrate proteins destined for secretion, including NUCB1, are phosphorylated by FAMC1 while in the Golgi apparatus. Due to an error in the script that parses our data to generate a GAF, that reaction was used to write rows to the GAF in which not only FAMC1 but also each of its substrates was tagged with protein-modification biological process terms. That script error has now been fixed so these annotations ot substrates should disappear with our next release in about a month.

On the larger question of what NUCB1 does do, with or without phosphorylation, we have no clue either.
